### DISCUSSION TITLE
Update url-parse to >1.5.6

### DISCUSSION THREAD

**Discussion by andrepaulo-bit** [wix/react-native-ui-lib#1858] [General]
`npm audit` shows that url-parse <1.5.6 has a vulnerability marked as High: https://github.com/advisories/GHSA-rqff-837h-mm52

This vulnerability has been fixed in 1.5.6

**Comment by ethanshar** [comment]
Hi @andrepaulo-bit 
Technically we use `"url-parse": "^1.2.0"` so we actually getting the version you mention (and even higher, `1.5.10`)
Are you sure it's an issue?
